isis/src/base/objs/ProcessImport/ProcessImport.h

```diff
@@ -163,11 +163,9 @@ namespace Isis {
    *                           was made to accomodate Rosetta VIRTIS-m calibrated data files and
    *                           has no impact on other supported BIP files.
    *                           Fixes #5398.
-   *   @history 2018-07-19 Tyler Wilson - Added support for 4-byte UnsignedInteger special pixel
+   *   @history 208-07-19 Tyler Wilson - Added support for 4-byte UnsignedInteger special pixel
```

Comment:
?

Reply:
Was removed due to me clobbering the changes I had made to the ProcessImport files. WIll re-add

Reply:
:+1: